website/src/components/Loading.tsx

```diff
@@ -0,0 +1,13 @@
+import CircularProgress from '@mui/material/CircularProgress';
+import Box from '@mui/material/Box';
+
+export function Loading() {
+
+    return (
+        <Box component="div" m={4} display="flex" flexDirection="column" justifyContent="center" alignItems="center" minHeight="50vh" >
+            <Box mb={5}>Loading...</Box>
+            <CircularProgress color="error" />
```

Comment:
As the software color is blue in general, I'd rather have the loading animation in blue as well.
```suggestion
            <CircularProgress color="primary" />
```

Reply:
yes, it had it set to error because I had adjusted that before adding the Error element.